### PR TITLE
Alerts if a package has been retired in hex.info

### PR DESCRIPTION
**Pull request by supersimple**
prints an alert when info is searched on a package version.
screen shot 2018-01-07 at 11 07 46 am

### CODE REVIEW

File: lib/mix/tasks/hex.info.ex [hunk 1]
```diff
@@ -153,6 +156,20 @@ defmodule Mix.Tasks.Hex.Info do
     "~> #{major}.#{minor}.#{patch}#{format_pre(pre)}"
   end
 
+  defp print_retirement(nil), do: ""
+  defp print_retirement(%{"retirement" => nil}), do: ""
+  defp print_retirement(release) do
+    reason = release["retirement"]["reason"]
+    message = release["retirement"]["message"]
+    Hex.Shell.info [[:bright, "This version has been retired "], [:reset, format_retirement_info(reason, message)]]
```
Comment (wojtekmach) [inline lib/mix/tasks/hex.info.ex]: please use `Hex.Utils.package_retirement_status/1` to get the message, see: https://github.com/hexpm/hex/blob/v0.17.1/lib/mix/tasks/hex.audit.ex#L48
 
maybe let's also use the yellow color as in `hex.audit`?

Comment (supersimple) [reply lib/mix/tasks/hex.info.ex]: Thanks. Just pushed up those changes.
screen shot 2018-01-07 at 12 49 45 pm

File: lib/mix/tasks/hex.info.ex [hunk 2]
```diff
@@ -153,6 +156,12 @@ defmodule Mix.Tasks.Hex.Info do
     "~> #{major}.#{minor}.#{patch}#{format_pre(pre)}"
   end
 
+  defp print_retirement(nil), do: ""
```
Comment (ericmj) [inline lib/mix/tasks/hex.info.ex]: We don't need this clause.

**Comment by wojtekmach** [comment]
Thank you!

**Pull request merged**